src/ng/compile.js

```diff
@@ -2170,6 +2172,20 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
     }
 
 
+    function downloadTemplateDependencies(directive) {
+      if (directive.directives) {
+        directive.directives.forEach(function(name) {
```

Comment:
the name `directives` sounds too generic... but this can be changed if this is moving forward